.github/workflows/main.yml

```
@@ -0,0 +1,35 @@
+name: Gradle (build & test)
+
+on:
+  push:
+    branches: &branches
+    - 'main'
+    - '1.16'
+    - '1.17'
+    - '1.18'
+    - '1.19'
+  pull_request:
+    branches: *branches
```

```
The workflow is not valid. .github/workflows/main.yml:
Anchors are not currently supported. Remove the anchor 'branches'
```

I guess GitHub doesn't like yaml anchors...
```suggestion
    branches:
    - 'main'
    - '1.16'
    - '1.17'
    - '1.18'
    - '1.19'
```